### PR TITLE
Xorg fontutil recipe

### PR DESCRIPTION
**Pull request by miqlas**
Just for fun

### CODE REVIEW

File: media-fonts/fontutil/fontutil-1.3.1.recipe [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+2007 Joerg Sonnenberger"
+LICENSE="MIT"
+SOURCE_URI="https://www.x.org/archive/individual/font/font-util-$portVersion.tar.gz"
+CHECKSUM_SHA256="34ebb0c9c14e0a392cdd5ea055c92489ad88d55ae148b2f1cfded0f3f63f2b5b"
+REVISION="1"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+SOURCE_DIR="font-util-$portVersion"
+
+#PATCHES="font-util--$portVersion.patchset"
```
Comment (fbrosson) [inline media-fonts/fontutil/fontutil-1.3.1.recipe]: The recommended order (and indentation) is:
```
SUMMARY="X.Org font utilities"
DESCRIPTION="X.Org font package creation/installation utilities"
HOMEPAGE="http://cgit.freedesktop.org/xorg/font/util"
COPYRIGHT="2006 Martin Husemann
	2007 Joerg Sonnenberger"
LICENSE="MIT"
REVISION="1"
SOURCE_URI="https://www.x.org/archive/individual/font/font-util-$portVersion.tar.gz"
CHECKSUM_SHA256="34ebb0c9c14e0a392cdd5ea055c92489ad88d55ae148b2f1cfded0f3f63f2b5b"
SOURCE_DIR="font-util-$portVersion"
#PATCHES="font-util--$portVersion.patchset"

ARCHITECTURES="!x86_gcc2 x86 x86_64"
SECONDARY_ARCHITECTURES="x86"
```
BTW, the last line is missing a newline. That's commonly found on MS Windows and that's not really an error on Unix-like systems and similar, but it's better to have a newline.

**Comment by miqlas** [comment]
I make my recipes on Haiku in Pe.

**Comment by fbrosson** [comment]
Shame on me, because I was suspecting that you were using Windows. I'm glad you were not ;-)

**Comment by waddlesplash** [comment]
I don't think this goes in media-fonts, but I don't have Gentoo Portage open to check atm.

**Comment by miqlas** [comment]
I checked 2 times.

-- Z.

2016.07.22. dátummal, 18:50 időpontban waddlesplash <notifications@github.com> írta:

**Pull request merged**